script.js

```diff
@@ -0,0 +1,201 @@
+
+
+// Functions goes here 👇
+
+// A function that will add a chat bubble in the correct place based on who the sender is
+const showMessage = (message, sender) => {
+  // The if statement checks if the sender is the user and if that's the case it inserts
+  // an HTML section inside the chat with the posted message from the user
+  if (sender === 'user') {
+    chat.innerHTML += `
+      <section class="user-msg">
+        <div class="bubble user-bubble">
+          <p>${message}</p>
+        </div>
+        <img src="user.png" alt="User" />  
+      </section>
+    `
+    // The else if statement checks if the sender is the bot and if that's the case it inserts
+    // an HTML section inside the chat with the posted message from the bot
+  } else if (sender === 'bot') {
+    // add a console.log here to see when it's being logged and not
+    console.log("Message is:", message);
+    console.log("Sender is:", sender);
+    chat.innerHTML += `
+      <section class="bot-msg">
+        <img src="bot.png" alt="Bot" />
+        <div class="bubble bot-bubble">
+          <p>${message}</p>
+        </div>
+      </section>
+    `
+  }
+
+  // This little thing makes the chat scroll to the last message when there are too many to
+  // be shown in the chat box
+  chat.scrollTop = chat.scrollHeight
+
+  // Clear the input field after submission
+  locationInput.value = ''
+}
+
+// A function to start the conversation
+const greetUser = () => {
+  showMessage("Hello, I can help you choose the perfect outfit based on the weather. What's your location?📍", 'bot')
+}
+
+// A function for the location input
+const handleLocationInput = (event) => {
+  event.preventDefault() // Prevent from refreshing the page
+  const userLocation = locationInput.value
+
+  if (userLocation) {
+    showMessage(userLocation, 'user')
+
+
+    // Remove the event listener for location input after being hanlded
+    locationForm.removeEventListener('submit', handleLocationInput);
+
+    // Checking weather...
+    setTimeout(() => {
+      showMessage(`Checking the weather for ${userLocation}...`, 'bot')
+
+      setTimeout(() => {
+        const weather = "sunny☀️ and between 18-20 degrees"
+        showMessage(`It looks like it's going to be ${weather} today in ${userLocation}.`, 'bot')
+        setTimeout(showClothingOptions, 1000)
+      }, 2000)
+    }, 1000)
+  }
+}
+
+// Function to create buttons for choosing casual or formal clothing
+const showClothingOptions = () => {
+  locationForm.removeEventListener('submit', handleInvalidInput);
+
+  chat.innerHTML += `
+    <section class="bot-msg clothing-options">
+      <img src="assets/bot.png" alt="Bot" />
+      <div class="bubble bot-bubble">
+        <p>Would you prefer casual or formal clothing?</p>
+        <button class="clothing-choice" data-choice="casual">Casual</button>
+        <button class="clothing-choice" data-choice="formal">Formal</button>
+      </div>
+    </section>
+  `
```

Comment:
You already have the showMessage to handle this! If you want buttons, use the input wrapper instead.